### DISCUSSION TITLE
How to load Time in Cube ?

### DISCUSSION THREAD

**Discussion by GameTn** [atoti/atoti#539] [Q&A]
Hi everyone , i have a dataframe contains time and i want to load the time to session ,every date has multiple times for exemple date 25/8/2010  have 12 time rows  ,but i only receive the last time not the full data 
![image](https://user-images.githubusercontent.com/101182803/160036002-489b17bd-cef5-4241-a6c8-acd670b63589.png)
 i have 4 tables : 
`Table Fact ` have 2 keys `id_DateandTime` (`type=Date` ,` Format=Day/Month/Year`) , and ` id_employee` (`type=string`) and measure `Salary `
`Table Employee `= ` id_employee` (`type=string`) , `FullName``(type=string)` ,`Department``(type=string)`
`Table Date`=`id_DateandTime` (`type=Date` ,` Format=Day/Month/Year`) , `Year` , `month` ,` day` all of them are string 
`Table Time` =` id_employee` (`type=string`) ,`id_DateandTime` (`type=Date` ,` Format=Day/Month/Year`) ,` Time` (`type=String` ,format is like this : 07:53:08 , 07:50:00 , 07:08:00)
i did this , 
`table Fact.join(Table Employee , mapping={"id_employee": "id_employee"})`
`Table Fact.join(Table Date , mapping={"id_DateandTime": "id_DateandTime"})`
`Table Fact.join(Table Time , mapping={"id_DateandTime": "id_DateandTime"})`
 Thanks

**Comment by bloussou** [comment]
Hello,
Do you have set your date column as *key* of the atoti `Table` and not the time column? Can you provide your column names and types and also the way you create your table? It is hard to help you without this information.

**Reply by GameTn** [reply]
Hi bloussou , its edited
